src/js/components/award/financialAssistance/RectanglePercentViz.jsx

```diff
@@ -25,102 +23,286 @@ const tooltipShape = PropTypes.shape({
+    const showNumeratorTooltip = (e) => {
+        e.stopPropagation();
+        showTooltip(numeratorTooltipData, "numerator");
+    };
+    const showDenominatorTooltip = (e) => {
+        e.stopPropagation();
+        showTooltip(denominatorTooltipData, "denominator");
+    };
+    const showNumerator2Tooltip = (e) => {
+        e.stopPropagation();
+        showTooltip(numerator2TooltipData, "numerator2");
+    };
+
+    const showNumerator3Tooltip = (e) => {
+        e.stopPropagation();
+        showTooltip(numerator3TooltipData, "numerator3");
+    };
+
+    const numerator3Positioning = {
+        position: 'relative',
+        right: relativeWidths.numerator2.width,
+        padding: '0.2rem'
+    };
+
     return (
         <div className="award-amounts-viz">
-            {!numeratorIsZero && <>
-                <div
-                    className="award-amounts-viz__desc-top--loans"
-                    role="button"
-                    tabIndex="0"
-                    onBlur={closeTooltip}
-                    onFocus={showNumeratorTooltip}
-                    onKeyPress={showNumeratorTooltip}
-                    onMouseEnter={showNumeratorTooltip}
-                    onMouseLeave={closeTooltip}
-                    onClick={showNumeratorTooltip}>
-                    <strong>{numeratorValue}</strong><br />{numerator.text}
-                </div>
+            {isTooltipVisible && <TooltipWrapper
+                className="award-section-tt"
+                onMouseMoveTooltip={setIsTooltipVisible.bind(null, true)}
+                onMouseLeaveTooltip={setIsTooltipVisible.bind(null, false)}
+                controlledProps={{
+                    isControlled: true,
+                    isVisible: true
+                }}
+                {...activeTooltipProps} />}
+            <div
+                className="award-amounts-viz__desc-top loans"
+                role="button"
+                tabIndex="0"
+                onBlur={closeTooltip}
+                onFocus={showNumeratorTooltip}
+                onKeyPress={showNumeratorTooltip}
+                onMouseEnter={showNumeratorTooltip}
+                onMouseLeave={closeTooltip}
+                onClick={showNumeratorTooltip}>
+                <strong>{numeratorValue}</strong>
+                <br />
+                {numerator.text}
+            </div>
+            {!numeratorIsZero &&
                 <div className="award-amounts-viz__label" style={numeratorBarAndLabelStyles}>
                     <div className="award-amounts-viz__line-up--loans" />
                 </div>
-            </>}
-            {numeratorIsZero &&
-                <TooltipWrapper {...numeratorTooltipProps} styles={{ width: '160px' }}>
-                    <div className="award-amounts-viz__desc-top--loans" role="button" tabIndex="0">
-                        <strong>{numeratorValue}</strong>
-                        <span>{numerator.text}</span>
+            }
+            {isCaresReleased &&
```

Comment:
Once cares is released do we still want to show this if it is zero? Because `isNumerator2Defined` does not handle this case.

Reply:
Good catch. Thanks Jon. 3deba28